torch/_export/__init__.py

```diff
@@ -106,34 +110,90 @@ def _export(
 
     with torch._dynamo.config.patch(dataclasses.asdict(ExportDynamoConfig())):  # type: ignore[attr-defined]
         try:
-            gm, _ = torch._dynamo.export(
+            gm_torch_level, _ = torch._dynamo.export(
                 f,
                 *args,
-                aten_graph=True,
-                tracing_mode="symbolic",
-                decomposition_table=DECOMP_TABLE,
                 constraints=constraints,
                 assume_static_by_default=True,
-                functionalize=True,
             )
+
+            params = OrderedDict()
+            buffers = OrderedDict()
+
+            for name, param in gm_torch_level.named_parameters(recurse=True, remove_duplicate=False):
+                params[name] = param
+
+            for name, param in gm_torch_level.named_buffers(recurse=True, remove_duplicate=False):
+                buffers[name] = param
+
+            fake_inps = []
+            for node in gm_torch_level.graph.nodes:
+                if node.op == "placeholder" and "val" in node.meta:
+                    fake_val = node.meta["val"]
+                    fake_inps.append(fake_val)
+
+            fake_mode = detect_fake_mode(fake_inps)
+
+            fake_args = pytree.tree_map_only(torch.Tensor, fake_mode.from_tensor, args)
```

Comment:
Probably not needed as we are making every placeholder entry to have fake tensor but just to be safe.